tooling-docker/src/test/java/org/jboss/eap/qe/ts/common/docker/DockerTest.java

```diff
@@ -0,0 +1,42 @@
+package org.jboss.eap.qe.ts.common.docker;
+
+import io.restassured.RestAssured;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import static org.hamcrest.Matchers.containsString;
+
+public class DockerTest {
+
+    @ClassRule
```

Comment:
What happens if I specify `"jboss/wildfly:0.0.0.0.Final"` ? Non-existing version case ...

I would expect error with clear message. There should be test for it ;)

Reply:
Just tried it with version `18.0.0.00':
```Starting container wildfly with ID 52670cb9-b4c2-4ca8-a1fc-8ece09432a72
wildfly> Unable to find image 'jboss/wildfly:18.0.0.00' locally
wildfly> docker: Error response from daemon: manifest for jboss/wildfly:18.0.0.00 not found.
wildfly> See 'docker run --help'.
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 60.072 s <<< FAILURE! - in org.jboss.eap.qe.ts.common.docker.DockerTest
[ERROR] org.jboss.eap.qe.ts.common.docker.DockerTest  Time elapsed: 60.072 s  <<< ERROR!
java.lang.Exception: Container was not ready in timeout : 60000 ms
	at org.jboss.eap.qe.ts.common.docker.ContainerReadyCondition.waitUntilReady(ContainerReadyCondition.java:17)
	at org.jboss.eap.qe.ts.common.docker.Docker.start(Docker.java:80)
	at org.jboss.eap.qe.ts.common.docker.Docker.before(Docker.java:103)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:46)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
```